exoctk/lightcurve_fitting/fitters.py

```diff
@@ -83,7 +91,7 @@ def lmfitter(time, data, model, unc=None, method='leastsq', verbose=True):
 
     # Make a new model instance
     best_model = copy.copy(model)
-    best_model.name = 'Best Fit'
     best_model.parameters = params
+    best_model.name = ', '.join(['{}:{}'.format(k,round(v[0], 2)) for k,v in params.dict.items()])
```

Comment:
@hover2pi Technically according to PEP8 this should be with some extra whitespace:

```python
best_model.name = ', '.join(['{}:{}'.format(k, round(v[0], 2)) for k, v in params.dict.items()])
```

Reply:
You caught me. I didn't flake8 it!